src/server/pipeTransport.ts

```diff
@@ -51,7 +49,7 @@ export class PipeTransport implements ConnectionTransport {
   }
 
   close() {
-    this._closeCallback();
+    // nothing to close with a pipe connection.
```

Comment:
Throw an Error as we shouldn't never be calling it on pipe?

Reply:
Done.